kodi_addon_checker/check_py3_compatibility.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+from lib2to3 import refactor
+from tabulate import tabulate
+
+from .report import Report
+from .record import Record, INFORMATION
+
+
+class KodiRefactoringTool(refactor.RefactoringTool):
+
+    def __init__(self, report, *args, **kwargs):
+        self.report = report
+        super(KodiRefactoringTool, self).__init__(*args, **kwargs)
+
+    def short_path(self, path):
+        return os.path.split(path)[1]
+
+    def line_num_for_phrase(self, filepath, phrase):
+        """
+        Return a dictionary having line number of the existing
+        code.
+            :filepath: Path of the file
+            :phrases: List of all the phrases to look for
+
+            :return: A dictionary of type
+                    {'existing_line_of_code': 'line-no'}
+        """
+
+        self.output = {}
+        for p in phrase:
+            with open(filepath, 'r') as f:
+                for (index, line) in enumerate(f):
+                    if p in line:
+                        self.output[p] = (index + 2)
+
+        return self.output
+
+    def print_output(self, old, new, filepath, equal):
+        """
+        Called with the old version, new version, and filepath of a
+        refactored file.
+
+            :old: old text of refactored file
+            :new: new text of refactored file
+            :filepath: Path of the file
+            :equal: Tells whether or not old is equal to new
+        """
+        tabulate.WIDE_CHARS_MODE = False
+        self.headers = ['File', 'Line-No', 'Existing Code', 'Changes required']
+        self.table = []
+
+        if equal:
+            return
+
+        if bool(set(new).intersection(old)):  # Tell whether there is some diff or not
+            new_changes = set(new.splitlines()) - set(old.splitlines())  # Give the requried changes
+            existing_line = set(old.splitlines()) - set(new.splitlines())  # Gives existing line
+            existing_code = self.line_num_for_phrase(filepath, list(existing_line))
+
+            for index, change in enumerate(sorted(existing_code)):
+                self.table.append([self.short_path(filepath), existing_code[change], change,
+                                   sorted(new_changes)[index]])
+
+        self.output = tabulate(self.table, headers=self.headers, tablefmt='fancy_grid')
+        self.report.add(Record(INFORMATION, self.output))
```

Comment:
`, '\n' + self.output))` so that first line of the table isn't indented by `INFO: `